catalog/transformer/catalog-transformer-service-xslt/src/main/java/ddf/catalog/services/xsltlistener/AbstractXsltTransformer.java

```diff
@@ -38,6 +38,7 @@
     try {
       unknownMime = new MimeType("application/octet-stream");
     } catch (MimeTypeParseException e) {
+      // do nothing
```

Comment:
❗️ Do we really want to do nothing here? This means that the DEFAULT_MIME_TYPE would be null now.If this is not expected to ever happen, then we should throw something like an InternalError or an IllegalStateException instead.